src/conn/conn_compact.c

```diff
@@ -249,6 +252,15 @@ __background_compact_should_run(WT_SESSION_IMPL *session, const char *uri, int64
         return (false);
     }
 
+    /* Fast path to check the file size. Ignore small files. */
+    filename = uri;
+    WT_PREFIX_SKIP(filename, "file:");
+    ret = __wt_block_manager_named_size(session, filename, &file_size);
+    if (ret == 0 && file_size <= WT_MEGABYTE) {
```

Comment:
I think we should do something if `ret` is not `0`:

1. At least print a log
2. Convert the function to return `int` and return the error. The current return value could become a parameter.

Reply:
done